tests/unit/preprocessor/_area/test_area.py

```diff
@@ -385,6 +420,22 @@ def test_crop_cube(make_testcube, square_shape, tmp_path):
         np.testing.assert_array_equal(result.data, expected)
 
 
+def test_crop_cube_with_ne_file():
+    """Test for cropping a cube by shape bounds."""
+    shp_file = "esmvalcore/preprocessor/ne_masks/ne_50m_ocean.shp"
+    with fiona.open(shp_file) as geometries:
```

Comment:
Actually this test could be done completely without the shapefile, but since you've already merged the pull request I guess that can wait for another time ;-)

Reply:
yes I know - the thing is I want to use it in #319 that's why I didn't change it here